spdk-sys/build.rs

```diff
@@ -79,6 +79,8 @@ fn main() {
         .whitelist_function("^spdk.*")
         .whitelist_function("create_malloc_disk")
         .whitelist_function("delete_malloc_disk")
+        .whitelist_function("bdev_null_create")
+        .whitelist_function("bdev_null_delete")
```

Comment:
The next whitelist entry should already cover these.

Reply:
done, good catch thanks :+1: